src/main/java/net/sourceforge/plantuml/servlet/DiagramResponse.java

```diff
@@ -96,6 +103,43 @@ public DiagramResponse(HttpServletResponse res, FileFormat fmt, HttpServletReque
         request = req;
     }
 
+    /**
+     * Initialize PlantUML configurations and properties as well as loading the PlantUML config file.
+     */
+    public static void init() {
+        if (initialized) {
+            return;
+        }
+        initialized = true;
+        // set allow include to false by default
+        OptionFlags.ALLOW_INCLUDE = false;
+        if ("true".equalsIgnoreCase(System.getenv("ALLOW_PLANTUML_INCLUDE"))) {
+            OptionFlags.ALLOW_INCLUDE = true;
+        }
+        // set security profile to INTERNET by default
+        // NOTE: this property is cached inside PlantUML and cannot be changed after the first call of PlantUML
+        System.setProperty("PLANTUML_SECURITY_PROFILE", SecurityProfile.INTERNET.toString());
+        if (System.getenv("PLANTUML_SECURITY_PROFILE") != null) {
+            System.setProperty("PLANTUML_SECURITY_PROFILE", System.getenv("PLANTUML_SECURITY_PROFILE"));
+        }
```

Comment:
**Breaking Change**
Setting the security profile to `INTERNET` should be the default.
But it is a breaking change and changing it should be well considered.